xsd/netex_framework/netex_reusableComponents/netex_serviceRestrictions_support.xsd

```diff
@@ -876,5 +878,37 @@ See ERA B.4 7037 Characteristic description code.</xsd:documentation>
 			</xsd:restriction>
 		</xsd:simpleContent>
 	</xsd:complexType>
+	<!-- ===== UserTypes============================================== -->
+	<xsd:simpleType name="UserTypeListOfEnumerations">
+		<xsd:annotation>
+			<xsd:documentation>List of user types.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:list itemType="UserTypeEnumeration"/>
+	</xsd:simpleType>
+	<xsd:simpleType name="UserTypeEnumeration">
+		<xsd:annotation>
+			<xsd:documentation>Allowed values for USER TYPE.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="xsd:normalizedString">
+			<xsd:enumeration value="adult"/>
+			<xsd:enumeration value="child"/>
+			<xsd:enumeration value="infant"/>
+			<xsd:enumeration value="senior"/>
+			<xsd:enumeration value="student"/>
+			<xsd:enumeration value="youngPerson"/>
+			<xsd:enumeration value="schoolPupil"/>
+			<xsd:enumeration value="military"/>
+			<xsd:enumeration value="veteran"/>
+			<xsd:enumeration value="disabled"/>
+			<xsd:enumeration value="disabledCompanion"/>
```

Comment:
Note that this is wrong in the UML diagram (disabledCompanon). 
Also: would it make sense to add *resident* to the list?

Reply:
It would that makes it more prominently so that this list is only of informational purpose. A relationship to a UserProfile is hard to make. For example how would we know this is a resident of a certain city, area or country?

Reply:
added